moto/transcribe/models.py

```diff
@@ -0,0 +1,401 @@
+            self.vocabulary_state = "READY"
+            self.last_modified_time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+
+
+class TranscribeBackend(BaseBackend):
+    def __init__(self, region_name=None):
+        self.medical_transcriptions = {}
+        self.medical_vocabularies = {}
+        self.region_name = region_name
+
+    def reset(self):
+        region_name = self.region_name
+        self.__dict__ = {}
+        self.__init__(region_name)
+
+    def start_medical_transcription_job(self, **kwargs):
+
+        name = kwargs.get("medical_transcription_job_name")
+
+        if name in self.medical_transcriptions:
+            raise RESTError(
+                error_type="ConflictException",
+                message="The requested job name already exists. Use a different job name.",
+                template="error_json",
+            )
+
+        settings = kwargs.get("settings")
+        vocabulary_name = settings.get("VocabularyName") if settings else None
+        if vocabulary_name and vocabulary_name not in self.medical_vocabularies:
+            raise RESTError(
+                error_type="BadRequestException",
+                message="The requested vocabulary couldn't be found. Check the vocabulary name and try your request again.",
+                template="error_json",
+            )
+
+        transcription_job_object = FakeMedicalTranscriptionJob(
+            region_name=self.region_name,
+            medical_transcription_job_name=name,
+            language_code=kwargs.get("language_code"),
+            media_sample_rate_hertz=kwargs.get("media_sample_rate_hertz"),
+            media_format=kwargs.get("media_format"),
+            media=kwargs.get("media"),
+            output_bucket_name=kwargs.get("output_bucket_name"),
+            output_encryption_kms_key_id=kwargs.get("output_encryption_kms_key_id"),
+            settings=settings,
+            specialty=kwargs.get("specialty"),
+            type=kwargs.get("type"),
+        )
+
+        self.medical_transcriptions[name] = transcription_job_object
+
+        return transcription_job_object.response_object("CREATE")
+
+    def get_medical_transcription_job(self, medical_transcription_job_name):
+        try:
+            job = self.medical_transcriptions[medical_transcription_job_name]
+            job.advance_job_status()  # Fakes advancement through statuses.
+            return job.response_object("GET")
+        except KeyError:
+            raise RESTError(
+                error_type="BadRequestException",
+                message="The requested job couldn't be found. Check the job name and try your request again.",
+                template="error_json",
+            )
+
+    def delete_medical_transcription_job(self, medical_transcription_job_name):
+        try:
+            del self.medical_transcriptions[medical_transcription_job_name]
+        except KeyError:
+            raise RESTError(
+                error_type="BadRequestException",
```

Comment:
Let's define these errors as stand-alone classes in a separate exceptions.py-file.
Ideally we only define the template-argument once in an parent, transcribe-wide, error class.

Reply:
Done.